### PR TITLE
modified pipeline doc folder exclude rule

### PR DESCRIPTION
**Pull request by bizz001**
### Which issue this PR addresses:

Fixes CI not ignoring commits containing only doc changes.

### What this PR does / why we need it:

Prevents CI from running on doc only commits.

### CODE REVIEW

**Review by petrkotas — APPROVED** [review]
LGTM nice catch

**Review by bryanro92 — CHANGES_REQUESTED** [review]
I'm not sure why - but this isn't working. CI is still running, and the pipelines run off your PR branch. I tested a similar change for e2e testing.

Could you implement what I did here for `e2e.yml`? https://github.com/bryanro92/ARO-RP/commit/d99a7a15aedacda3602d6f23d0c2b0c8d30c0bc0

I copied exact pattern from ci.yml to e2e.yml and it circumvented e2e from running, but CI is still running :thinking: 

EDIT: CI actually blew up on my suggested changes with
```
ruamel.yaml.scanner.ScannerError: mapping values are not allowed here
  in ".pipelines/e2e.yml", line 2, column 8
```
However, if we get CI to exclude properly then it wont fail :smile:

File: .pipelines/ci.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ trigger:
     - master
   paths:
     exclude:
-    - docs/*
+    - docs/
```
Comment (m1kola) [inline .pipelines/ci.yml:8]: It should probably be a recursive wildcard instead: `**`. Found some docs on it:

* https://docs.microsoft.com/en-us/azure/devops/release-notes/2021/sprint-192-update#support-for-wild-cards-in-path-filters
* https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml
* https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/file-matching-patterns?view=azure-devops

**Comment by m1kola** [comment]
@bizz001 is it something we still need? If so, could you please rebase and make sure that CI is green?

**Comment by cadenmarchese** [comment]
@bizz001 bump to address comments above (if we still need this PR).

**Pull request closed without merging**